linalg/hypre.hpp

```diff
@@ -1394,6 +1426,11 @@ class HypreParaSails : public HypreSolver
 
    virtual void SetOperator(const Operator &op);
 
+   void SetParams(double threshold, int max_levels);
```

Comment:
Thanks for adding all these additional HYPRE parameter setters.  Just the other day someone was asking me for these.

Reply:
Indeed, thanks for these @ptsuji !